libs/MeshKernelApi/tests/src/CurvilinearGridTests.cpp

```diff
@@ -795,3 +795,51 @@ TEST_F(CartesianApiTestFixture, CurvilinearDeleteInterior_OnCurvilinearGrid_Shou
     ASSERT_NEAR(meshkernel::constants::missing::doubleValue, curvilinearGrid.node_x[72], tolerance);
     ASSERT_NEAR(meshkernel::constants::missing::doubleValue, curvilinearGrid.node_y[72], tolerance);
 }
+
+TEST(CurvilinearGrid, MakeRectangular_ComputeSmoothnessTest)
```

Comment:
small comment: MakeRectangular_ComputeSmoothnessTest. For the test name, i try to follow the convention: MethodName_StateUnderTest_ExpectedBehavior. See the test above this